aliyun_driver.go

```diff
@@ -190,6 +191,10 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 			DefaultString: "ManagedKubernetes",
 		},
 	}
+	driverFlag.Options["kubernetes-version"] = &types.Flag{
```

Comment:
Existing aliyun clusters (before this update) will have this field passed as an empty string (`""`) on update operations.  Will this cause errors or otherwise mess up the cluster? Should we set a default k8s version here so that the correct value is passed?

Reply:
1. When an empty string is passed, Aliyun provision a cluster with the default version.
2. So I think it is proper to have an empty string as the default value for this field.